src/Activation/AcceptEula.tsx

```diff
@@ -70,6 +70,10 @@ const AcceptEula: FunctionComponent = () => {
     loadEula()
   }, [eulaPath])
 
+  const handleOnPressNext = () => {
+    navigation.navigate(ActivationScreens.ActivateProximityTracing)
```

Comment:
Does this want to navigate to the next screen in the stack somehow? Instead of having this screen hardcoded?

Reply:
Same with the navigation from `ActivateProximityTracing` to the next screen.

Reply:
I see where you're coming from @devinjameson you mean like having a navigate next property on each one of the onboarding/activation screen props? and then we could call navigate with the next screen prop? If this is owned by each screen and it's reused I would be onboard.

Reply:
sounds like a great idea to me.

Reply:
though i do want to be careful not to over engineer this. We are working with react-navigation after all, we likely wont be able to get to only one call site for routing given the lack of a schema.